runtime.js

```diff
@@ -14,11 +16,16 @@ function checkProcessArgv (moduleName) {
 
 let preloadModules
 function checkPreloadModules (moduleName) {
+  const modulePath = path.join(process.cwd(), 'node_modules/', moduleName)
```

Comment:
Not sure about this path join.
I just want to do a `startsWith` instead of `includes`.

Reply:
Can `process.cwd()` guarantee to be the project root if executed from other path?

Reply:
What would you do?